frontend/src/pages/Skills/SkillsContainer.tsx

```diff
@@ -16,7 +16,7 @@ const SkillsContainer: FC = () => {
   const { data: skillsResponse } = useGetSkills();
 
   useEffect(() => {
-    if (debouncedSearchValue) {
+    if (debouncedSearchValue && debouncedSearchValue.length >= 2) {
```

Comment:
magic number

Reply:
and should we check both of these conditions? or maybe debouncedSearchValue.length will be enough?